.gitignore

```diff
@@ -109,4 +109,4 @@ share/python-wheels/
 *.egg-info/
 .installed.cfg
 *.egg
-MANIFEST
```

Comment:
no

Reply:
IIRC this is automatically generated by the build process.